articles/quickstart/webapp/_includes/_getting_started.md

```diff
@@ -1,7 +1,7 @@
 <!-- markdownlint-disable MD041 -->
 
 ::: note
-**New to Auth?** Learn [How Auth0 works](/overview), how it [integrates with Regular Web Applications](/architecture-scenarios/application/web-app-sso) and which [protocol](/flows/concepts/auth-code) it uses.
+**New to Auth?** Learn [How Auth0 works](/overview), how it [integrates with Regular Web Applications](/architecture-scenarios/web-app-sso) and which [protocol](/flows/concepts/auth-code-pkce) it uses.
```

Comment:
A lot of the Web App SDKs use `code`, perhaps we should just link to `/flows/concepts`

Reply:
`/flows/concepts` doesn't exist, changing to `/flows`.

Reply:
Changed in https://github.com/auth0/docs/pull/9557/commits/e932a625798bedc2b280ff2606c57b780ac4ef15.